### ISSUE TITLE
Regarding scale

### ISSUE THREAD

**Issue by rounak-mehta** [flyyufelix/DenseNet-Keras#15]
Can you explain why you're using a separate scale function when batch norm scales the output anyway?

**Comment by flyyufelix** [comment]
You can refer to this thread: https://github.com/flyyufelix/DenseNet-Keras/issues/5
